keywords/couchbaseserver.py

```diff
@@ -337,20 +354,27 @@ def _get_mem_total_lowest(self, server_info):
                 # only use it if it's lower than previous low
                 mem_total_lowest = mem_total
 
-        if mem_total_lowest is None:
-            raise ProvisioningError("All nodes reported 0MB of RAM available")
+        """if mem_total_lowest is None:
```

Comment:
we need this change I think

Reply:
what change? I commented out the change, I did not remove. Once server team fix the ram issue, we can uncomment the change and revert the retries.